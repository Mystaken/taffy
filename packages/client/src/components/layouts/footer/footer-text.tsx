import React, { FunctionComponent } from 'react';
import { TypographyProps, Typography } from '@material-ui/core';

export const FooterText: FunctionComponent<TypographyProps> = props => (
  <Typography {...props} color="textSecondary"></Typography>
);
