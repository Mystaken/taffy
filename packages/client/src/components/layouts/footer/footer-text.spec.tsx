import React from 'react';
import { create } from 'react-test-renderer';

import { FooterText } from './footer-text';

describe('<FooterText />', () => {
  it('has rendered correctly', () => {
    const component = create(<FooterText />);
    const tree = component.toJSON()!;
    expect(tree).toMatchSnapshot();
  });
});
