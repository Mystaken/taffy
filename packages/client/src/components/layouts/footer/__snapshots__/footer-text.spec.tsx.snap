// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterText /> has rendered correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
/>
`;
